oeps/oep-0001.rst

```diff
@@ -300,111 +368,84 @@ ReStructuredText [8] allows for rich markup that is relatively easy to read,
 and can also be rendered into good-looking and functional HTML. OEPs are
 rendered to HTML using Sphinx. An `OEP template`_ can be found in the repo.
 
-OEPs may be discussed in a more convenient format, such as a Google Doc, if
-it is deemed appropriate for the audience and sufficiently open to comment and
-review. The final OEP must be transcribed into the `reStructuredText`_-based
-template and committed to the OEP repository. The Arbiter shall be responsible
-for ensuring the proper transcription. The reviewed and accepted OEP must
-reference the location of relevant discussion, and the ownership of the
-discussion document should be transferred to the edX Chief Architect, if
-applicable.
-
 .. _reStructuredText: http://docutils.sourceforge.net/rst.html
-.. _OEP template: https://github.com/cpennington/open-edx-proposals/blob/master/oep-template.rst
+.. _OEP template: https://github.com/edx/open-edx-proposals/blob/master/oep-template.rst
 
 OEP Header Preamble
 -------------------
 
-Each OEP must begin with an ReST table with metadata about the OEP. The rows
+Each OEP must begin with a ReST table with metadata about the OEP. The rows
```

Comment:
I personally would prefer "an RST table".  I would read "ReST" or "reST" as "rest", which makes me think about the wrong thing.  Wikipedia claims all 3 are used: https://en.wikipedia.org/wiki/ReStructuredText